WARN  [risk-p2p-server] 2017-05-18 09:07:27.078 [main] org.springframework.context.support.PropertySourcesPlaceholderConfigurer - Could not load properties from class path resource [conf/jms.properties]: class path resource [conf/jms.properties] cannot be opened because it does not exist
ERROR [risk-p2p-server] 2017-05-18 10:30:50.861 [DubboServerHandler-172.16.88.107:10020-thread-58] cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl - ********************** Exception is thrown **********************
异常编号 : EXCEPTION_CODE_016679605591260
产生异常的类名 : cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl
产生异常的方法名 : selectTodoList
方法参数 : [cn.sunfit.risk.buz.api.beans.p2p.activiti.TodoQueryReq@9e63658]
异常提示信息 : 在业务层发生了异常,可能是一个BUG,请将日志文件中异常编号为[EXCEPTION_CODE_016679605591260]的异常明细发送给开发团队
异常发生的时间 : 2017-05-18 10:30:50
异常堆栈信息 : orj.worf.exception.AppRuntimeException: nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
### The error may exist in file [D:\workspace\risk-parent\risk-server\risk-p2p-server\target\classes\sqlmap\p2p\activiti\BP_SqlMap.xml]
### The error may involve cn.sunfit.risk.buz.server.dao.p2p.activiti.BPDAO.selectTodoList-Inline
### The error occurred while setting parameters
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78)
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55)
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85)
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33)
	at sun.reflect.GeneratedMethodAccessor202.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653)
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl$$EnhancerBySpringCGLIB$$1178bf30.selectTodoList(<generated>)
	at com.alibaba.dubbo.common.bytecode.Wrapper26.invokeMethod(Wrapper26.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
### The error may exist in file [D:\workspace\risk-parent\risk-server\risk-p2p-server\target\classes\sqlmap\p2p\activiti\BP_SqlMap.xml]
### The error may involve cn.sunfit.risk.buz.server.dao.p2p.activiti.BPDAO.selectTodoList-Inline
### The error occurred while setting parameters
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:231)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy24.selectList(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:205)
	at org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:117)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy62.selectTodoList(Unknown Source)
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl.selectTodoList(BpServiceImpl.java:94)
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl$$FastClassBySpringCGLIB$$4fe457dc.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58)
	... 42 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400)
	at com.mysql.jdbc.Util.getInstance(Util.java:383)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:980)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2002)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:430)
	at orj.worf.mybatis.util.CountHelper.getCount(CountHelper.java:42)
	at orj.worf.mybatis.interceptor.PaginationInterceptor.intercept(PaginationInterceptor.java:52)
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:60)
	at com.sun.proxy.$Proxy46.prepare(Unknown Source)
	at org.apache.ibatis.executor.ReuseExecutor.prepareStatement(ReuseExecutor.java:76)
	at org.apache.ibatis.executor.ReuseExecutor.doQuery(ReuseExecutor.java:56)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	at sun.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 56 more


ERROR [risk-p2p-server] 2017-05-18 10:30:50.868 [DubboServerHandler-172.16.88.107:10020-thread-58] com.alibaba.dubbo.rpc.filter.ExceptionFilter -  [DUBBO] Got unchecked and undeclared exception which called by 172.16.88.107. service: cn.sunfit.risk.buz.api.service.p2p.activiti.BpService, method: selectTodoList, exception: orj.worf.exception.AppRuntimeException: nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
### The error may exist in file [D:\workspace\risk-parent\risk-server\risk-p2p-server\target\classes\sqlmap\p2p\activiti\BP_SqlMap.xml]
### The error may involve cn.sunfit.risk.buz.server.dao.p2p.activiti.BPDAO.selectTodoList-Inline
### The error occurred while setting parameters
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause', dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
orj.worf.exception.AppRuntimeException: nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
### The error may exist in file [D:\workspace\risk-parent\risk-server\risk-p2p-server\target\classes\sqlmap\p2p\activiti\BP_SqlMap.xml]
### The error may involve cn.sunfit.risk.buz.server.dao.p2p.activiti.BPDAO.selectTodoList-Inline
### The error occurred while setting parameters
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33) ~[worf-service-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.GeneratedMethodAccessor202.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl$$EnhancerBySpringCGLIB$$1178bf30.selectTodoList(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at com.alibaba.dubbo.common.bytecode.Wrapper26.invokeMethod(Wrapper26.java) ~[na:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:na]
	at java.lang.Thread.run(Thread.java:745) [na:na]
Caused by: org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
### The error may exist in file [D:\workspace\risk-parent\risk-server\risk-p2p-server\target\classes\sqlmap\p2p\activiti\BP_SqlMap.xml]
### The error may involve cn.sunfit.risk.buz.server.dao.p2p.activiti.BPDAO.selectTodoList-Inline
### The error occurred while setting parameters
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:231) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at com.sun.proxy.$Proxy24.selectList(Unknown Source) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:205) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:117) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy62.selectTodoList(Unknown Source) ~[na:na]
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl.selectTodoList(BpServiceImpl.java:94) ~[classes/:na]
	at cn.sunfit.risk.buz.server.service.p2p.activiti.BpServiceImpl$$FastClassBySpringCGLIB$$4fe457dc.invoke(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	... 42 common frames omitted
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'base.loancar_license_plate' in 'where clause'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:na]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:na]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.Util.getInstance(Util.java:383) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:980) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2002) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:430) ~[c3p0-0.9.5-pre9.jar:0.9.5-pre9]
	at orj.worf.mybatis.util.CountHelper.getCount(CountHelper.java:42) ~[worf-mybatis-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.mybatis.interceptor.PaginationInterceptor.intercept(PaginationInterceptor.java:52) ~[worf-mybatis-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:60) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy46.prepare(Unknown Source) ~[na:na]
	at org.apache.ibatis.executor.ReuseExecutor.prepareStatement(ReuseExecutor.java:76) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.ReuseExecutor.doQuery(ReuseExecutor.java:56) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.GeneratedMethodAccessor228.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358) ~[mybatis-spring-1.2.2.jar:1.2.2]
	... 56 common frames omitted
WARN  [risk-p2p-server] 2017-05-18 11:33:17.666 [main] org.springframework.context.support.PropertySourcesPlaceholderConfigurer - Could not load properties from class path resource [conf/jms.properties]: class path resource [conf/jms.properties] cannot be opened because it does not exist
WARN  [risk-p2p-server] 2017-05-18 13:29:30.588 [New I/O worker #1] com.alibaba.dubbo.remoting.transport.AbstractServer -  [DUBBO] All clients has discontected from /172.16.88.107:10020. You can graceful shutdown now., dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
WARN  [risk-p2p-server] 2017-05-18 13:36:56.253 [New I/O worker #2] com.alibaba.dubbo.remoting.transport.AbstractServer -  [DUBBO] All clients has discontected from /172.16.88.107:10020. You can graceful shutdown now., dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
WARN  [risk-p2p-server] 2017-05-18 14:27:44.496 [main] org.springframework.context.support.PropertySourcesPlaceholderConfigurer - Could not load properties from class path resource [conf/jms.properties]: class path resource [conf/jms.properties] cannot be opened because it does not exist
ERROR [risk-p2p-server] 2017-05-18 16:32:34.302 [DubboServerHandler-172.16.88.107:10020-thread-134] cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl - ********************** Exception is thrown **********************
异常编号 : EXCEPTION_CODE_0128381841566782
产生异常的类名 : cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl
产生异常的方法名 : saveRole
方法参数 : [cn.sunfit.risk.buz.api.vo.corp.CorpRoleAddReq@3bebd81a]
异常提示信息 : 在业务层发生了异常,可能是一个BUG,请将日志文件中异常编号为[EXCEPTION_CODE_0128381841566782]的异常明细发送给开发团队
异常发生的时间 : 2017-05-18 16:32:34
异常堆栈信息 : orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78)
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55)
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85)
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33)
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>)
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy24.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58)
	... 42 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400)
	at com.mysql.jdbc.Util.getInstance(Util.java:383)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62)
	at com.sun.proxy.$Proxy46.update(Unknown Source)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 59 more


ERROR [risk-p2p-server] 2017-05-18 16:32:34.313 [DubboServerHandler-172.16.88.107:10020-thread-134] com.alibaba.dubbo.rpc.filter.ExceptionFilter -  [DUBBO] Got unchecked and undeclared exception which called by 172.16.88.107. service: cn.sunfit.risk.buz.api.service.corp.CorpRoleService, method: saveRole, exception: orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY', dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33) ~[worf-service-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java) ~[na:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:na]
	at java.lang.Thread.run(Thread.java:745) [na:na]
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at com.sun.proxy.$Proxy24.insert(Unknown Source) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source) ~[na:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65) ~[classes/:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	... 42 common frames omitted
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'f6c7773a75a74f8f82c791d01a4928e3-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:na]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:na]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.Util.getInstance(Util.java:383) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404) ~[c3p0-0.9.5-pre9.jar:0.9.5-pre9]
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy46.update(Unknown Source) ~[na:na]
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358) ~[mybatis-spring-1.2.2.jar:1.2.2]
	... 59 common frames omitted
ERROR [risk-p2p-server] 2017-05-18 16:32:49.274 [DubboServerHandler-172.16.88.107:10020-thread-136] cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl - ********************** Exception is thrown **********************
异常编号 : EXCEPTION_CODE_0128396891354051
产生异常的类名 : cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl
产生异常的方法名 : saveRole
方法参数 : [cn.sunfit.risk.buz.api.vo.corp.CorpRoleAddReq@1126e9f6]
异常提示信息 : 在业务层发生了异常,可能是一个BUG,请将日志文件中异常编号为[EXCEPTION_CODE_0128396891354051]的异常明细发送给开发团队
异常发生的时间 : 2017-05-18 16:32:49
异常堆栈信息 : orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
; SQL []; Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78)
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55)
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85)
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33)
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>)
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
; SQL []; Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy24.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58)
	... 42 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400)
	at com.mysql.jdbc.Util.getInstance(Util.java:383)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62)
	at com.sun.proxy.$Proxy46.update(Unknown Source)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 59 more


ERROR [risk-p2p-server] 2017-05-18 16:32:49.277 [DubboServerHandler-172.16.88.107:10020-thread-136] com.alibaba.dubbo.rpc.filter.ExceptionFilter -  [DUBBO] Got unchecked and undeclared exception which called by 172.16.88.107. service: cn.sunfit.risk.buz.api.service.corp.CorpRoleService, method: saveRole, exception: orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
; SQL []; Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY', dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
; SQL []; Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33) ~[worf-service-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java) ~[na:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:na]
	at java.lang.Thread.run(Thread.java:745) [na:na]
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
; SQL []; Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at com.sun.proxy.$Proxy24.insert(Unknown Source) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source) ~[na:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65) ~[classes/:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	... 42 common frames omitted
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '1d744025cfac42c1918c6cb0f2139518-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:na]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:na]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.Util.getInstance(Util.java:383) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404) ~[c3p0-0.9.5-pre9.jar:0.9.5-pre9]
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy46.update(Unknown Source) ~[na:na]
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358) ~[mybatis-spring-1.2.2.jar:1.2.2]
	... 59 common frames omitted
ERROR [risk-p2p-server] 2017-05-18 16:33:36.741 [DubboServerHandler-172.16.88.107:10020-thread-137] cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl - ********************** Exception is thrown **********************
异常编号 : EXCEPTION_CODE_0128444355486349
产生异常的类名 : cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl
产生异常的方法名 : saveRole
方法参数 : [cn.sunfit.risk.buz.api.vo.corp.CorpRoleAddReq@1563ac18]
异常提示信息 : 在业务层发生了异常,可能是一个BUG,请将日志文件中异常编号为[EXCEPTION_CODE_0128444355486349]的异常明细发送给开发团队
异常发生的时间 : 2017-05-18 16:33:36
异常堆栈信息 : orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78)
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55)
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85)
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33)
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>)
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy24.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65)
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58)
	... 42 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400)
	at com.mysql.jdbc.Util.getInstance(Util.java:383)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62)
	at com.sun.proxy.$Proxy46.update(Unknown Source)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 59 more


ERROR [risk-p2p-server] 2017-05-18 16:33:36.745 [DubboServerHandler-172.16.88.107:10020-thread-137] com.alibaba.dubbo.rpc.filter.ExceptionFilter -  [DUBBO] Got unchecked and undeclared exception which called by 172.16.88.107. service: cn.sunfit.risk.buz.api.service.corp.CorpRoleService, method: saveRole, exception: orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY', dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
orj.worf.exception.AppRuntimeException: nested exception is org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at orj.worf.exception.util.ExceptionUtils.createAppRuntimeException(ExceptionUtils.java:78) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.validateException(ExceptionInterceptor.java:55) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at orj.worf.exception.interceptor.ExceptionInterceptor.handleAppException(ExceptionInterceptor.java:44) ~[worf-exception-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:603) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:62) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:85) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at orj.worf.service.aspect.ServiceLogAspect.logInboundOutbound(ServiceLogAspect.java:33) ~[worf-service-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at sun.reflect.GeneratedMethodAccessor219.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:621) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:610) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:68) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$EnhancerBySpringCGLIB$$1170c9c9.saveRole(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at com.alibaba.dubbo.common.bytecode.Wrapper2.invokeMethod(Wrapper2.java) ~[na:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:48) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:77) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:57) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:58) ~[worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:43) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:79) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:57) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:106) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:39) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:40) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:100) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:114) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:88) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:178) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:53) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:86) [worf-remoting-dubbo-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:na]
	at java.lang.Thread.run(Thread.java:745) [na:na]
Caused by: org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
### The error may involve cn.sunfit.risk.buz.server.dao.corp.CorpRoleDAO.insertRoleFunc-Inline
### The error occurred while setting parameters
### SQL: insert into   risk_corp_role_func_rel(role_id,func_id,create_time) values         (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )    ,     (?,    ?,    now()    )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
; SQL []; Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:239) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73) ~[spring-jdbc-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at com.sun.proxy.$Proxy24.insert(Unknown Source) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:240) ~[mybatis-spring-1.2.2.jar:1.2.2]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy34.insertRoleFunc(Unknown Source) ~[na:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl.saveRole(CorpRoleServiceImpl.java:65) ~[classes/:na]
	at cn.sunfit.risk.buz.server.service.corp.CorpRoleServiceImpl$$FastClassBySpringCGLIB$$67b4e015.invoke(<generated>) ~[spring-core-4.1.3.RELEASE.jar:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:52) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:267) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	at org.springframework.aop.aspectj.AspectJAfterThrowingAdvice.invoke(AspectJAfterThrowingAdvice.java:58) ~[spring-aop-4.1.3.RELEASE.jar:4.1.3.RELEASE]
	... 42 common frames omitted
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'c27c4547681840b2ae73e1ad11087f6d-143' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:na]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526) ~[na:na]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:400) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.Util.getInstance(Util.java:383) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3847) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3783) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2447) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2594) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2545) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1901) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1193) ~[mysql-connector-java-5.1.36.jar:5.1.36]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:404) ~[c3p0-0.9.5-pre9.jar:0.9.5-pre9]
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.GeneratedMethodAccessor260.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:62) ~[mybatis-3.2.8.jar:3.2.8]
	at com.sun.proxy.$Proxy46.update(Unknown Source) ~[na:na]
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152) ~[mybatis-3.2.8.jar:3.2.8]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141) ~[mybatis-3.2.8.jar:3.2.8]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:na]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358) ~[mybatis-spring-1.2.2.jar:1.2.2]
	... 59 common frames omitted
WARN  [risk-p2p-server] 2017-05-18 17:32:25.825 [New I/O worker #1] com.alibaba.dubbo.remoting.transport.AbstractServer -  [DUBBO] All clients has discontected from /172.16.88.107:10020. You can graceful shutdown now., dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
WARN  [risk-p2p-server] 2017-05-18 17:34:43.483 [New I/O worker #2] com.alibaba.dubbo.remoting.transport.AbstractServer -  [DUBBO] All clients has discontected from /172.16.88.107:10020. You can graceful shutdown now., dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
WARN  [risk-p2p-server] 2017-05-18 20:53:53.812 [New I/O worker #3] com.alibaba.dubbo.remoting.transport.AbstractServer -  [DUBBO] All clients has discontected from /172.16.88.107:10020. You can graceful shutdown now., dubbo version: 0.1.0-SNAPSHOT, current host: 127.0.0.1
